src/utils/array_utils.F90.in

```diff
@@ -5,22 +5,24 @@ module array_utils
   ! This module contains routines for working with arrays.
   !
   ! NOTE(wjs, 2015-10-21) These routines could be moved to csm_share. However, I'm not
-  ! sure if these routines will stick around for long, since they are just here to work
-  ! around the fact that we don't specify -Mallocatable=03 with pgi (which in turn is
+  ! sure if some of these routines will stick around for long, since they are just here to
+  ! work around the fact that we don't specify -Mallocatable=03 with pgi (which in turn is
```

Comment:
There was a new utility subroutine added "find_k_max_indices" that likely might stay around. So this note should say that's only true for the two subroutines that replicate the two intrinsics.

Reply:
Done.